### DISCUSSION TITLE
Pomysły nowych funkcji Miarkownika01

### DISCUSSION THREAD

**Discussion by krzychoooo** [krzychoooo/Miarkownik#5] [Ideas]
**Tu zgłaszamy pomysły nowych funkcji Miarkownika01**

**Comment by piotrux** [comment]
Witam, proszę o dodanie funkcji montażu z prawej lub lewej strony pieca, tj. aby 0 było 180 a 180 było 0 przy montażu z prawej strony. Aktualnie miarkownik można podłączyć tylko z lewej strony pieca. 
Pozdrawiam i kibicuję

**Comment by piotrux** [comment]
Proszę o dodanie przy temperaturze nastawianej ikonki plusa i minusa, ułatwi to szybkie ustawianie temperatury na piecu o np 2, 3 lub 5 'C, aktualnie trzeba klinka na cyfry i z "palca" klepać nowa temperaturę.

**Reply by krzychoooo** [reply]
Zrobione w Termostat01b_2020_12_16_17_25.apk

**Reply by piotrux** [reply]
Super, to jest to.
Jeszcze ikonki wifi, paliwo i dźwignia na prawą stronę bo zaśmiecają zegar temperatury :)

**Reply by AndyBoom** [reply]
Wydaje mi się, że gdyby w aplikacji skala była co 10 stopni byłoby przyjaźniej (np. od 0 do 100)

**Reply by krzychoooo** [reply]
Przy najbliższej pracy nad aplikacją wezmę to pod uwagę.

**Reply by AndyBoom** [reply]
Jeżeli dałoby radę, to warto byłoby dodać do aplikacji możliwość ustawienia alarmów dźwiękowych przy kilku zadanych temperaturach, np. przy 90 st., aby reagować zanim zagotuje się w piecu oraz np. przy ok. 50, gdyby użytkownik nie zauważył, że kończy mu się paliwo ;)

**Reply by krzychoooo** [reply]
Tak. Będzie już następnej wersji.

**Comment by krzychoooo** [comment]
Najbliższa przyszłość to:

1. Możliwość z poziomu aplikacji ustawienia 4 par (czas, temperatura), dzięki czemu można ustawić automatyczne zmiany temperatury w ciągu doby. np w nocy chłodniej w dzień cieplej.
2. Możliwość wysyłania aktualnej temperatury na zewnętrzny serwer MQTT np thinkspeek, aby można było podglądnąć jaka jest temperatura na kotle z poza domu.

Doba ma skończoną ilość godzin więc pytanie co najpierw?

**Reply by piotrux** [reply]
Dla mnie 1

**Reply by AndyBoom** [reply]
Dla mnie też pozycja nr 1

**Comment by piotrux** [comment]
Jak wszytko zadziała, jak trzeba, mam zamiar sparować miarkownik z netatmo, czy macie jakieś doświadczenie z taką funkcjonalnością, będzie potrzebny np domoticz to oczywiste ale czy konfiguracja i kalibracja będzie czasochłonna?

**Reply by krzychoooo** [reply]
Miarkownik jest już przygotowany do współpracy z domoticz,. 
U mnie już działa od 2 `miesięcy 
![Screenshot_2020-12-17 Dom](https://user-images.githubusercontent.com/7173366/102548434-f09f5280-40ba-11eb-973c-b4e140b05369.png)

Niedługo wideo-poradnik jak w domoticzu zrobić termostat sterujący miarkownikiem.
A netamo też można spiąć z domoticzem więc sprawa będzie raczej prosta.
